fmriprep/workflows/bold/util.py

```diff
@@ -87,7 +87,8 @@ def init_bold_reference_wf(omp_nthreads, bold_file=None, name='bold_reference_wf
 First, a reference volume and its skull-stripped version were generated
 using a custom methodology of *fMRIPrep*.
```

Comment:
Can we update the boilerplate description with a conditional? That will likely require you to add a new argument to the function, and with it maybe avoid the re-run of validation.

I think it is relevant to write that fMRIPrep used the SBRef to the boilerplate.

Reply:
For this workflow, the `sbref_file` is a workflow input, not a function input. That makes making it conditional somewhat difficult.

Reply:
What's the expected boilerplate if some series have sbref and others don't?